mod-gobi/schemas/mapping.json

```diff
@@ -28,6 +30,7 @@
         "DISCOUNT",
         "DISCOUNT_TYPE",
         "DONOR",
+        "DONOR_DEPRECATED",
```

Comment:
In story acceptance criterias written that we need support derecated field poLine.donor for some transition period and in next R release this filed will be deleted